pages/clustering/high-availability.mdx

```diff
@@ -68,6 +68,17 @@ use bolt+routing in different programming languages [here](https://github.com/me
 It is important to note that setting up the cluster on one coordinator (registration of data instances and coordinators, setting main) must be done using bolt connection 
 since bolt+routing is only used for routing data-related queries, not coordinator-based queries.
 
+## System configuration
+
+<Callout type="warning">
+Important note if you're using native Memgraph deployment with RedHat.
+
+Redhat uses SELinux to enforce security policies.
+SELinux (Security-Enhanced Linux) is a security mechanism for implementing mandatory access control (MAC) in the Linux kernel.
+It restricts programs, users, and processes to only the resources they require, following a least-privilege model.
+When deploying Memgraph with high availability (HA), consider checking out this attribute for instance visibility.
```

Comment:
What exactly user should do? (checking the attribute? can you be more specific)

Reply:
check now